From: wangyixue <wangyixue@uniontech.com>
Date: Fri, 3 Feb 2023 10:06:29 +0800
Subject: upgrade cmake minumum version require

Upgrade cmake minumum version require to suppress warnings from ECM.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f0c25f..0c6112e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.6.0)
+cmake_minimum_required(VERSION 3.16.0)
 project(fcitx VERSION 5.0.23)
 set(FCITX_VERSION ${PROJECT_VERSION})
 
